distribution/resources/conf/default-micro-gw.conf.template

```diff
@@ -504,6 +504,15 @@
   timestampSkew = 5000
   # The value of the server header sent in the response by microgateway
   header="ballerina"
+  # Preserve or override header values intercepted by the gateway
+  [[server.header]]
+      headerName = "User-Agent"
+      preserveHeader = true
+      overrideValue = "ballerina"
+  [[server.header]]
+      headerName = "Server"
+      preserveHeader = true
```

Comment:
The default behaviour is we override this value. So default vales should be false right?

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/1577/commits/4bdc97f8146428c3a0e2b6355d8a316b06724651